swagger/v1-prerelease/swagger.yaml

```diff
@@ -0,0 +1,1950 @@
+        element:
+          type: integer
+          format: int32
+          readOnly: true
+        card:
+          type: integer
+          format: int32
+          readOnly: true
+        mask:
+          type: integer
+          format: int32
+          readOnly: true
+      additionalProperties: false
+    DicomPrivateCreator:
+      type: object
+      properties:
+        creator:
+          type: string
+          nullable: true
+      additionalProperties: false
+    DicomTag:
+      type: object
+      properties:
+        group:
+          type: integer
+          format: int32
+        element:
+          type: integer
+          format: int32
+        isPrivate:
+          type: boolean
+          readOnly: true
+        privateCreator:
+          $ref: '#/components/schemas/DicomPrivateCreator'
+        dictionaryEntry:
+          $ref: '#/components/schemas/DicomDictionaryEntry'
+      additionalProperties: false
+    DicomVM:
+      type: object
+      properties:
+        minimum:
+          type: integer
+          format: int32
+        maximum:
+          type: integer
+          format: int32
+        multiplicity:
+          type: integer
+          format: int32
+      additionalProperties: false
+    DicomVR:
+      type: object
+      properties:
+        code:
+          type: string
+          nullable: true
+          readOnly: true
+        name:
+          type: string
+          nullable: true
+          readOnly: true
+        isString:
+          type: boolean
+          readOnly: true
+        isStringEncoded:
+          type: boolean
+          readOnly: true
+        is16bitLength:
+          type: boolean
+          readOnly: true
+        isMultiValue:
+          type: boolean
+          readOnly: true
+        paddingValue:
+          type: integer
+          format: int32
+          readOnly: true
+        maximumLength:
+          type: integer
+          format: int32
+          readOnly: true
+        unitSize:
+          type: integer
+          format: int32
+          readOnly: true
+        byteSwap:
+          type: integer
+          format: int32
+          readOnly: true
+        valueType:
+          type: string
+          nullable: true
+          readOnly: true
+      additionalProperties: false
+    ExtendedQueryTagStatus:
+      enum:
+        - 0
+        - 1
+        - 2
+      type: integer
+      format: int32
+    GetExtendedQueryTagEntry:
+      type: object
+      properties:
+        status:
+          $ref: '#/components/schemas/ExtendedQueryTagStatus'
+        level:
+          $ref: '#/components/schemas/QueryTagLevel'
+        path:
+          type: string
+          nullable: true
+        vr:
+          type: string
+          nullable: true
+        privateCreator:
+          type: string
+          nullable: true
+      additionalProperties: false
+    QueryTagLevel:
+      enum:
+        - 0
+        - 1
+        - 2
+      type: integer
+      format: int32
+    Stream:
```

Comment:
This pesky type is still here lol. Perhaps left over after the operation filters cleaned up its references?